packages/link/src/utils/popup.ts

```diff
@@ -101,6 +102,18 @@ const getStyles = (style?: LinkStyle) => `
 </style>
 `
 
+function addPlatformSpecs(): void {
```

Comment:
you are adding variables to parent window and window for the parent of the parent. We need to add sdk version to  iframe window. Please use `postMessage` and post new event to the iframe window like in https://github.com/FrontFin/front-b2b-link/blob/main/packages/link/src/FrontConnection.ts#L90
Also you need to handle that event on catalog page at front-web-platform repo